src/main/java/org/traccar/MainModule.java

```diff
@@ -340,8 +341,16 @@ public static SpeedLimitHandler provideSpeedLimitHandler(@Nullable SpeedLimitPro
     @Provides
     public static BroadcastService provideBroadcastService(
             Config config, ObjectMapper objectMapper) throws IOException {
+        String broadcastType = config.getString(Keys.BROADCAST_TYPE);
         if (config.hasKey(Keys.BROADCAST_ADDRESS)) {
-            return new MulticastBroadcastService(config, objectMapper);
+            switch (broadcastType) {
```

Comment:
I think it makes sense to just do switch on `config.getString(Keys.BROADCAST_TYPE)` without checking `BROADCAST_ADDRESS`.

Reply:
Hi, I didn't want to introduce a breaking change in here, on Keys made BROADCAST_TYPE default to "multicast" to match current behavior, with that in mind would you either prefer to remove BROADCAST_TYPE default and BROADCAST_ADDRESS check or switch's default? thanks.

Reply:
Let's do a breaking change. It's a relatively new feature, so should be fine.